README.md

```
@@ -1 +1,224 @@
-# Otboo [![codecov](https://codecov.io/gh/sb11-code-rangers/sb11-otboo-team4/branch/dev/graph/badge.svg)](https://codecov.io/gh/sb11-code-rangers/sb11-otboo-team4)
\ No newline at end of file
+# Otboo (옷장을 부탁해) [![codecov](https://codecov.io/gh/sb11-code-rangers/sb11-otboo-team4/graph/badge.svg)](https://codecov.io/gh/sb11-code-rangers/sb11-otboo-team4)
+
+> 날씨·취향을 고려해 사용자가 보유한 의상 조합을 추천해주고,<br>
+> OOTD 피드·팔로우·DM 등의 소셜 기능을 갖춘 개인화 의상 추천 SaaS
+
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)